src/houghtransform.jl

```diff
@@ -0,0 +1,102 @@
+
+#function to compute local maximum lines with values > threshold and return a vector containing them
+function findlocalmaxima(accumulator_matrix::Array{Integer,2},threshold::Integer)
+    validLines = Vector{CartesianIndex}(0)
+    for val in CartesianRange(size(accumulator_matrix))
+        if  accumulator_matrix[val] > threshold &&
+            accumulator_matrix[val] > accumulator_matrix[val[1],val[2] - 1] &&
```

Comment:
`accumulator_matrix` has size `(numangle + 2, numrho + 2)`, basically applying a padding of width 1 over the entire image. While [filling the accumulator matrix](https://github.com/annimesh2809/ImageFeatures.jl/blob/d17d674a54b3f4b3f066e92e23e4b2456f30ae6a/src/houghtransform.jl#L76), we fill it at `(i+1,dist+1)` rather than `(i,dist)` which prevents any `BoundsError`.
![img](https://cloud.githubusercontent.com/assets/15063205/24140808/5764dd28-0e47-11e7-9c63-8b19a573c63c.png)